webapp/packages/plugin-navigation-tree/src/NavigationTree/ElementsTree/useElementsTree.ts

```diff
@@ -502,9 +502,9 @@ export function useElementsTree(options: IOptions): IElementsTree {
         return (options.filters || []).reduce((children, filter) => filter(elementsTree, elementsTree.filter, node, children, this.state), children);
       },
       async setFilter(value: string) {
-        runInAction(() => {
-          this.userData.filter = value;
+        this.userData.filter = value;
 
+        runInAction(() => {
           for (const nodeState of this.state.values()) {
             nodeState.showInFilter = false;
           }
```

Comment:
seems like case fix but not fix of the problem
actual problem is here:
https://github.com/dbeaver/cloudbeaver/blob/1105f1300a7fc6f8b0e4403c597cd74391cf929c/webapp/packages/plugin-navigation-tree/src/NavigationTree/ElementsTree/useElementsTree.ts#L203

explanation:
`loadTree` triggered concurrently with user change
`userData` updated but it state isn't applied yet